### PR TITLE
Add Camera Flashlight

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add `flashlight` component
- Adds a keybind to toggle a camera flashlight like the one in 3DEN

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Special thanks to the [ACE3 Team](http://ace3mod.com/team.html) for their open s
 - Vehicle customization garage made specifically for Zeus.
 - Various bug fixes and quality of life improvements to the Zeus interface.
 - Settings to control Zeus camera properties such as speed and available vision modes.
+- Zeus camera flashlight for easier nighttime editing.
```
Comment (CreepPork) [inline README.md]:
```suggestion
- Zeus camera flashlight for easier night-time editing.
```

File: docs/home.md [hunk 2]
```diff
@@ -23,4 +23,5 @@ Special thanks to the [ACE3 Team](http://ace3mod.com/team.html) for their open s
 - Vehicle customization garage made specifically for Zeus.
 - Various bug fixes and quality of life improvements to the Zeus interface.
 - Settings to control Zeus camera properties such as speed and available vision modes.
+- Zeus camera flashlight for easier nighttime editing.
```
Comment (CreepPork) [inline docs/home.md]:
```suggestion
- Zeus camera flashlight for easier night-time editing.
```

File: README.md (hunk 1 above)
Comment (mharis001) [reply README.md]: Both are correct, but "... for easier editing at night." might be best?

Comment (CreepPork) [reply README.md]: I'd prefer the one with the hyphen because it's easier to read, but your variant would work too.

Comment (neilzar) [reply README.md]: I'd go with mharis' one, sounds a bit nicer.

Comment (neilzar) [reply README.md]:
```suggestion
- Zeus camera flashlight for easier editing at night.
```

File: docs/home.md (hunk 2 above)
Comment (neilzar) [reply docs/home.md]:
```suggestion
- Zeus camera flashlight for easier editing at night.
```

**Pull request merged**